pkg/nfd-worker/nfd-worker.go

```diff
@@ -172,6 +172,39 @@ func newDefaultConfig() *NFDConfig {
 	}
 }
 
+func (w *nfdWorker) createTicker() *time.Ticker {
+	if w.config.Core.SleepInterval.Duration > 0 {
+		return time.NewTicker(w.config.Core.SleepInterval.Duration)
+	} else {
```

Comment:
This Else is not needed

Reply:
The else is now removed

Reply:
TO me this looks better like a  https://gobyexample.com/switch since the func doesn't have a `return`  a Switch will be more elegant here